sdk/python/featurestore_sample/project/fraud_model/online_inference/src/scoring.py

```diff
@@ -0,0 +1,73 @@
+import os
+import logging
+import json
+import time
+import pyarrow
+import pickle
+
+from azure.identity import ManagedIdentityCredential
+from azureml.featurestore import FeatureStoreClient
+from azureml.featurestore import init_online_lookup
+from azureml.featurestore import get_online_features
+
+
+def init():
+    """
+    This function is called when the container is initialized/started, typically after create/update of the deployment.
+    You can write the logic here to perform init operations like caching the model in memory
+    """
+
+    global model
+
+    # load the model
+    print("check model path")
+
+    model_path = os.path.join(os.getenv("AZUREML_MODEL_DIR"), "model_output/clf.pkl")
+
+    with open(model_path, "rb") as pickle_file:
+        model = pickle.load(pickle_file)
+    # AZUREML_MODEL_DIR is an environment variable created during deployment.
+    # It is the path to the model folder (./azureml-models/$MODEL_NAME/$VERSION)
+    # Please provide your model's folder name if there is one
+
+    # load feature retrieval spec
+    print("load feature spec")
+
+    credential = ManagedIdentityCredential()
+
+    spec_path = os.path.join(os.getenv("AZUREML_MODEL_DIR"), "model_output")
+
+    global features
+
+    featurestore = FeatureStoreClient(credential=credential)
+
+    features = featurestore.resolve_feature_retrieval_spec(spec_path)
+
+    init_online_lookup(features, credential)
+
+    time.sleep(20)
```

Comment:
nit: Why are we waiting here?

Reply:
As per comments elsewhere in a similar example _It may take ~10 sec or so for the local RPC server of online store client to be ready_.

Reply:
Yeah, but why the wait? `init_online_lookup` might take 10 seconds to return, but once it does, the server is initialized and ready to go.